### ISSUE TITLE
Dockerfile not working

### ISSUE THREAD

**Issue by Brutus5000** [FAForever/QAI#66]
There are some issues with building the Dockerfile due to dependecy issues.

**Comment by Brutus5000** [comment]
Still unresolved. But the error has changed:
```faf-qai                          | Traceback (most recent call last):
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/base.py", line 214, in include
faf-qai                          |     module = next(entry_points).load()
faf-qai                          | StopIteration
faf-qai                          |
faf-qai                          | During handling of the above exception, another exception occurred:
faf-qai                          |
faf-qai                          | Traceback (most recent call last):
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/bin/irc3", line 11, in <module>
faf-qai                          |     sys.exit(run())
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/__init__.py", line 405, in run
faf-qai                          |     return IrcBot.from_argv(argv)
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/base.py", line 449, in from_argv
faf-qai                          |     context = cls.from_config(cfg)
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/base.py", line 388, in from_config
faf-qai                          |     context = cls(**cfg)
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/__init__.py", line 119, in __init__
faf-qai                          |     super(IrcBot, self).__init__(*ini, **config)
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/base.py", line 115, in __init__
faf-qai                          |     self.include(*self.config.get('includes', []))
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/base.py", line 216, in include
faf-qai                          |     raise exc
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/base.py", line 206, in include
faf-qai                          |     module = utils.maybedotted(module)
faf-qai                          |   File "/root/.local/share/virtualenvs/code-_Py8Si6I/lib/python3.6/site-packages/irc3/utils.py", line 313, in maybedotted
faf-qai                          |     'Not able to resolve %s' % name)
faf-qai                          | LookupError: Not able to resolve qai_plugin
```

**Comment by Petricpwnz** [comment]
TL;DR - in the config file change qai_plugin to qai.qai_plugin, that should fix it.

config ini is supposed to end up 'botdirectory/' and the plugin file in 'botdirectory/qai/' hence "qai.qai_plugin". For non-docker, i dont know where you have the config file on the server but if you for some reason need it to be in some other directory besides the bot directory then would need to change the config accordingly

**Comment by norraxx** [comment]
How can I simulate such behavior?
I'm on Ubuntu 18.04, `Docker version 18.09.0, build 4d60db4`

**Comment by samuel-hxs** [comment]
If i am correct, the configuration is placed at [faf-stack](https://github.com/FAForever/faf-stack/blob/879b575c092d3a40971e95d9249901251b569f89/config.template/faf-qai/config.ini#L12) in the faf-stack project.
For me one other file had to be changed.

**Comment by Brutus5000** [comment]
Fixed in https://github.com/FAForever/faf-stack/pull/131
